src/main/java/nextstep/app/application/CustomOauth2UserService.java

```diff
@@ -0,0 +1,35 @@
+package nextstep.app.application;
+
+import nextstep.app.domain.Member;
+import nextstep.app.domain.MemberRepository;
+import nextstep.security.oauth2.OAuth2AuthenticationException;
+import nextstep.security.oauth2.OAuth2ProviderClient;
+import nextstep.security.oauth2.OAuth2UserInfo;
+import nextstep.security.oauth2.user.OAuth2UserRequest;
+import nextstep.security.oauth2.user.OAuth2UserService;
+import nextstep.security.oauth2.user.Oauth2User;
+import org.springframework.stereotype.Service;
+
+import java.util.Set;
+
+@Service
+public class CustomOauth2UserService implements OAuth2UserService {
```

Comment:
```suggestion
public class CustomOauth2UserService implements OAuth2UserService<OAuth2UserRequest, Oauth2User> {
```
로 표현되는 것이 조금 더 정확할 것 같아요 :)

Reply:
앗.. 그러네요! 제너릭을 빠뜨렸네요..

Reply:
[8f121af](https://github.com/next-step/spring-security-oauth2/pull/15/commits/8f121afafcdd8ee5e643fd911db43ee60e01e848) 해당 커밋에 반영 하였습니다.